mx-tester.yml

```diff
@@ -34,6 +35,9 @@ homeserver:
   enable_registration: true
   enable_registration_without_verification: true
 
+  app_service_config_files:
+    - "/data/mjolnir-registration.yaml"
```

Comment:
Don't forget to add this file to the PR :)

Reply:
It shouldn't be added to the PR, it needs to be generated from mx-tester.yml